packages/features/bookings/lib/constants.ts

```diff
@@ -0,0 +1 @@
+export const HARD_LIMIT_MAX_LENGTH = 4000;
```

Comment:
This is a limit for textarea only. How about we keep it in form-builder/fieldTypes as textarea.hardLimitOnMaxLength.

In this way it is better organized.

Reply:
I am taking care of it. I will play around and see what is best, as it's been a long time I worked on form builder

Reply:
Fair enough, can do.